config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/EntityUtils.java

```diff
@@ -78,15 +89,18 @@ private EntityUtils() {
         private FileType fileType;
         private String environmentType;
 
+        private boolean bundleGenerationSupported;
+
         private GatewayEntityInfo() {
         }
 
-        private GatewayEntityInfo(String type, Class<? extends GatewayEntity> entityClass, String fileName, FileType fileType, String environmentType) {
+        private GatewayEntityInfo(String type, Class<? extends GatewayEntity> entityClass, String fileName, FileType fileType, String environmentType, boolean bundleGenerationSupported) {
```

Comment:
I suppose bundleGenerationSupported is computable right inside. 
No need to make it as constructor-level arg.

Reply:
done